src/__main__.py

```diff
@@ -1,34 +1,63 @@
-from pathlib import Path
-from pprint import pprint as pp
+import os
 import sys
+from datetime import datetime, timedelta
+
 from .parse_pst import get_email
 
+FIELD = "subject"
+LAST_HOUR_TEST = datetime(2022, 7, 16, 18, 0, 0, 0) - timedelta(
+    hours=1
+)  # only used for dev purposes
+
+LAST_HOUR = datetime.now() - timedelta(hours=1)  # for 'real time' use
+
+
+def is_windows():
+    """
+    https://stackoverflow.com/a/1325587
+    """
+    return os.name == "nt"
+
+
+def open_text_file(text_file):
+    open_cmd = "start" if is_windows() else "open"
+    os.system(f"{open_cmd} {text_file}")  # noqa S605
 
-def email_digest(pst_file): 
+
+def email_digest(pst_file):
```

Comment:
I would break this into two functions: one to parse the pst, building up the data structure as you've done. Then another one to write it to an output file (and for that I would make an output folder you add to .gitignore), ok? Let me know if you have questions or hit roadblocks please.